src/time/span.cr

```diff
@@ -53,18 +53,21 @@ struct Time::Span
   # @nanoseconds can either be negative or positive).
   @nanoseconds : Int32
 
-  def self.new(hours : Int, minutes : Int, seconds : Int)
-    new(0, hours, minutes, seconds)
+  def self.new(days : Int, hours : Int, minutes : Int, seconds : Int, nanoseconds : Int)
+    new(
+      compute_seconds(days, hours, minutes, seconds),
+      nanoseconds.to_i64,
+    )
   end
 
-  def self.new(days : Int, hours : Int, minutes : Int, seconds : Int, nanoseconds : Int = 0)
+  def self.new(*, days : Int = 0, hours : Int = 0, minutes : Int = 0, seconds : Int = 0, nanoseconds : Int = 0)
     new(
-      seconds: compute_seconds!(days, hours, minutes, seconds),
-      nanoseconds: nanoseconds.to_i64,
+      compute_seconds(days, hours, minutes, seconds),
+      nanoseconds.to_i64,
     )
   end
 
-  def initialize(*, seconds : Int, nanoseconds : Int)
+  def initialize(seconds : Int, nanoseconds : Int)
```

Comment:
Please revert this. There's no reason to make these arguments positional.

Reply:
I am quite unsure how to deal with overloads, as having named arguments here leads to stack overflow

Reply:
Never mind, I just realized i misunderstood the docs 😓